scalekit/events.py

```
@@ -0,0 +1,54 @@
+            None
+        """
+        self.core_client = core_client
+        self.events_service = EventsServiceStub(
+            self.core_client.grpc_secure_channel
+        )
+
+    def list_events_paginated(
+        self,
+        page_size: Optional[int] = None,
+        page_token: Optional[str] = None,
+        filter: Optional[EventFilter] = None,
```

Ruff (0.15.21)

[error] Function argument `filter` is shadowing a Python builtin

(A002)